Dockerfile

```diff
@@ -1,26 +1,27 @@
 # Build the manager binary
-# Build the manager binary
-FROM golang:1.16 as builder
+FROM --platform=${BUILDPLATFORM} golang:1.16 as builder
+
+ARG TARGETARCH
+ARG TARGETOS
 
 WORKDIR /workspace
 # Copy the Go Modules manifests
-COPY go.mod go.mod
-COPY go.sum go.sum
+COPY go.mod go.sum ./
 # cache deps before building and copying source so that we don't need to re-download as much
 # and so that source changes don't invalidate our downloaded layer
 RUN go mod download
 
 # Copy the go source
-COPY main.go main.go
+COPY main.go ./
```

Comment:
I would rather be consistent here, for example we write
```Dockerfile
COPY api/ api/
```

But we could just as well do it this way.
```Dockerfile
COPY api/ .
```

Personally I like the `.` more but I think this is clearer and clarity is a good thing in open-source :), so I think we should keep the current way so please revert it.